### ISSUE TITLE
Aquile Reader for Android!

### ISSUE THREAD

**Issue by OptimiliaStudios** [OptimiliaStudios/AquileReader#126]
Aquile Reader for Android is now publicly available under open testing program (early access). Please try out the app and share your feedback.

[Play Store link](https://play.google.com/store/apps/details?id=com.optimiliastudios.aquilereader)

With this we bring **cross-device sync of books** functionality to both Android and Windows (in Insider Program) apps, so your books, reading state, highlights, notes and bookmarks are synced to cloud and available across devices for seamless reading experience.

The cross-device sync for Windows app is only available in Insider Program. To join, simply share the email address you use for logging into the Microsoft Store with us at optimilia.studios@gmail.com

**Comment by rahulTPatil** [comment]
Hello,

When trying the play store link it says the early access program is currently full. When is it going to be available for all users? Any ETA would be appreciated.

Thanks

**Comment by OptimiliaStudios** [comment]
The app is now in Open Testing, so you should be able to download now.

**Comment by Pixel123-web** [comment]
Pros: The UI is excellent, it's user friendly. Nice touch with the sync option and web access. 
Cons: it takes about 3 minutes to open a book, thereafter, it only loads one page, the other pages appear blank. It doesn't read PDF files.

**Comment by Ledang99** [comment]
> Pros: The UI is excellent, it's user friendly. Nice touch with the sync option and web access. Cons: it takes about 3 minutes to open a book, thereafter, it only loads one page, the other pages appear blank. It doesn't read PDF files.

Yes, having the same problem. Second book taking long time to load (spinning n spinning), therefore cancelling it.

Also please don't make a copy of book for annotate/notes etc as it will taking additional space.

**Comment by harsha3562** [comment]
Inbuilt dictionary is not working after the recent update

**Comment by maidmillennial** [comment]
Ive been using the app as my ebook reader, as well as I usually listen to the book being read. After so long of it reading outlook the app shuts down and I am constantly having to restart it. Ive gone into my settings thinking it stops when my screen goes to sleep, and changed the settings. But it continues to shut down after so long. Any suggestions?

**Comment by StephenCYang** [comment]
Question: I have a generic Chinese made Android 14.0 ereader. It has 32GB of storage. It supports a 1TB microSD card though. If I copy all my epubs to the SD card and then add those to my Aquile library, does it copy them to my device's storage or does it just keep the actual epub on the SD card and simply creates a entry (link and cover) in device storage?

If it just makes entries of title and link, I should be able to add all of them in one fell swoop?

**Comment by OptimiliaStudios** [comment]
No, the books are not copied again on the device storage, the app uses the eBook files from the source.
Yes, the app has a feature to bulk add books. Though the app does not copy files, it does some processing of books being added. So, when trying to add extremely large amount of Books in one go, you may experience some issues, hence it's suggested to add in batches.

**Comment by aryanbarnwal3575-a11y** [comment]
Where is scroll for page change ( continuous like as if scrolling a website option on the app ??? ) kindly help asap . It's an app of my dreams I just want where to enable that feature instead of page change by tapping
